modules/k8s-cluster/iam.tf

```diff
@@ -31,13 +31,12 @@ data "aws_iam_policy_document" "ssm-minimal" {
   }
 }
 
-data "aws_iam_policy_document" "ecr_access" {
```

Comment:
is losing the ecr policy intentional? :thinking:

Reply:
https://github.com/alphagov/gsp/pull/991/commits/49634c8766cd3e676ed6522f75f9258051a481be

Reply:
touché